### PR TITLE
Change ScoutingNano event content

### PR DESCRIPTION
**Pull request by patinkaew**
#### PR description:

This PR changes the event content of ScoutingNano to address the following points:
- drop scouting PF candidates to reduce size (https://github.com/cms-sw/cmssw/issues/45339)
- drop scouting tracks to reduce size (requested during previous presentation at XPOG, supported by latest study [1])
- add scouting PF jets (clustered at HLT)
    - scouting PF jets (clustered at HLT) has eta range extended to |eta| < 5 from |eta| < 3, starting in 2024. Currently, ScoutingNano only contain jets reclustered from scouting PF candidates which still have eta range |eta| < 3. Therefore, this allows ones to also access forward jets from scouting PF jets (clustered at HLT).
    - With this addition, all scouting objects collected at HLT (except PF candidates and tracks) are included. This is useful for regular object performance monitoring and trigger efficiency studies, for instance.
- remove tasks related to TrigObj table
    - Scouting objects are, by definition, trigger objects (TrigObj).
    - ```patTriggerScouting``` step in producing TrigObj table causes very noisy log (```trigger::TriggerEvent product with InputTag 'hltTriggerSummaryAOD::HLT' not in event``` for every event). This issue was raised by O&C (https://github.com/cms-sw/cmssw/pull/44970#issuecomment-2213462892). This is caused by missing ```hltTriggerSummaryAOD::HLT``` in ScoutingPFRun3 RAW/HLTSCOUT samples.
    - In fact, TrigObj table never gets produced because [triggerObjectTableScouting](https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/custom_run3scouting_cff.py#L48) is dropped from event content as specified in [NanoAODEventContent](https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/NanoAODEDMEventContent_cff.py#L6) as the producer doesn't end with "Table" suffix.
    - Hence, we consider removing TrigObj table and related tasks to solve noisy log file.
- minor fixes
    - change ```_lambda``` to ```lambda_``` for ```ScoutingTrack``` collection so the output branch name is ```ScoutingTrack_lambda_``` instead of ```Scouting__lambda``` in root file.
    - change branch names and docs for jets to clearly distinguish between jets clustered at HLT and reclustered ones.
    - fix typo particlenet_uds to particleNet_uds to match other particleNet branches in reclustered AK4 jets.
    - fix incorrect orders of particleNet branches in reclustered AK8 jets to match [preprocess.json](https://github.com/cms-data/RecoBTag-Combined/blob/master/Run3Scouting/ParticleNetAK8/General/V00/preprocess.json#L2-L6).

#### PR validation:

Event sizes are measured with relval_nano matrix for ScoutingNano: 
- 0.88 kb/event (2500.131 ScoutingPFRun32022RAW13.0+scoutingNANO_data13.0)
- 1.14 kb/event (2500.237 ScoutingPFRun32024RAW14.0+scoutingNANO_data14.0)
- 1.86 kb/event (2500.227 TTbarMINIAOD14.0+scoutingNANO_mc14.0)

For full size reports, please refer to [1].

In addition, this PR pass all standard unit tests ```scram b runtests```
```
Pass   13s ... PhysicsTools/NanoAOD/testPhysicsToolsNanoAODTP
```
and pass all tests from running ```runTheMatrix.py -l limited -i all --ibeos```
```
43 42 41 36 19 1 1 1 1 1 1 tests passed, 0 0 0 0 0 0 0 0 0 0 0 failed
```

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

This PR is not a backport, but backport to 14_0_X (together with https://github.com/cms-sw/cmssw/pull/44970) will follow.

FYI: @silviodonato @elfontan @karim

[1] https://indico.cern.ch/event/1441452/#preview:5119073

### CODE REVIEW

**Comment by hqucms** [comment]
enable nano

**Comment by hqucms** [comment]
please test

File: PhysicsTools/NanoAOD/python/run3scouting_cff.py [hunk 1]
```diff
@@ -141,7 +141,7 @@
          nValidStripHits = Var('tk_nValidStripHits', 'int', doc='number of valid strip hits'),
          nTrackerLayersWithMeasurement = Var('tk_nTrackerLayersWithMeasurement', 'int', doc='number of tracker layers with measurements'),
          qoverp = Var('tk_qoverp', 'float', precision=10, doc='qoverp'),
-         _lambda = Var('tk_lambda', 'float', precision=10, doc='lambda'),
+         lambda_ = Var('tk_lambda', 'float', precision=10, doc='lambda'),
```
Comment (vlimant) [inline PhysicsTools/NanoAOD/python/run3scouting_cff.py:144]: why keeping the trailing underscore at all ?

Comment (patinkaew) [reply PhysicsTools/NanoAOD/python/run3scouting_cff.py:144]: ```lambda``` is a keyword in python and I believe that was the reason initially to call it ```_lambda```.

File: PhysicsTools/NanoAOD/python/run3scouting_cff.py [hunk 2]
```diff
@@ -334,17 +363,17 @@
 
 ak4ScoutingJetTable = cms.EDProducer("SimplePFJetFlatTableProducer",
       src = cms.InputTag("ak4ScoutingJets"),
-      name = cms.string("ScoutingJet"),
+      name = cms.string("ScoutingCHSJetRecluster"),
       cut = cms.string(""),
-      doc = cms.string("ScoutingJet"),
+      doc = cms.string("ak4 jets from re-clustering scouting PF candidates with CHS applied"),
```
Comment (hqucms) [inline PhysicsTools/NanoAOD/python/run3scouting_cff.py:368]: Where is CHS applied to `ak4ScoutingJets` and `ak8ScoutingJets`? I see in https://github.com/cms-sw/cmssw/blob/ab08a21a7d4644a6e7c5e79f1857cd464b85cb51/PhysicsTools/Scouting/plugins/Run3ScoutingParticleToRecoPFCandidateProducer.cc#L281 the default value of `CHS` is set to `false`.

Comment (patinkaew) [reply PhysicsTools/NanoAOD/python/run3scouting_cff.py:368]: Thank you for catching this! There seems to be mismatch (whether CHS is applied or not) in private and cmssw code. I will rename them to PFJets.

**Comment by hqucms** [comment]
please test

**Comment by patinkaew** [comment]
Hi @hqucms,

Thank you for rerunning the tests! Checking the log files of failed tests, I think they are all because MINIAOD input files cannot be open.

- [2500.212_NANOdata140Xrun3/step2_NANOdata140Xrun3.log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-29a25c/41315/runTheMatrixNANO-results/2500.212_NANOdata140Xrun3/step2_NANOdata140Xrun3.log)
- [2500.232_EGMNANOdata140Xrun3/step2_EGMNANOdata140Xrun3.log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-29a25c/41315/runTheMatrixNANO-results/2500.232_EGMNANOdata140Xrun3/step2_EGMNANOdata140Xrun3.log)
- [2500.233_BTVNANOdata140Xrun3/step2_BTVNANOdata140Xrun3.log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-29a25c/41315/runTheMatrixNANO-results/2500.233_BTVNANOdata140Xrun3/step2_BTVNANOdata140Xrun3.log)
- [2500.234_jmeNANOdata140Xrun3/step2_jmeNANOdata140Xrun3.log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-29a25c/41315/runTheMatrixNANO-results/2500.234_jmeNANOdata140Xrun3/step2_jmeNANOdata140Xrun3.log)
- [2500.236_lepTrackInfoNANOdata140Xrun3/step2_lepTrackInfoNANOdata140Xrun3.log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-29a25c/41315/runTheMatrixNANO-results/2500.236_lepTrackInfoNANOdata140Xrun3/step2_lepTrackInfoNANOdata140Xrun3.log)

Checking DAS, the MINIAOD file seems to be on disk, so it was probably a temporary problem when tests were run.

**Comment by hqucms** [comment]
@smuzaffar It seems that this file disappeared?
```
/eos/cms/store/user/cmsbuild/store/data/Run2024D/MuonEG/MINIAOD/PromptReco-v1/000/380/306/00000/1cfada9f-5e3e-4f4c-b38b-f55455e2e9f5.root
```

**Comment by smuzaffar** [comment]
@hqucms , `/eos/cms/store/user/cmsbuild/store/data/Run2024D/MuonEG/MINIAOD/PromptReco-v1/000/380/306/00000/1cfada9f-5e3e-4f4c-b38b-f55455e2e9f5.root` is now available. Looks like it was never cached and we were always reading it from `root://xrootd-cms.infn.it` . I will check why it was not cached

**Comment by smuzaffar** [comment]
by the way, I have restarted the nano relval tests

**Comment by hqucms** [comment]
> @hqucms , `/eos/cms/store/user/cmsbuild/store/data/Run2024D/MuonEG/MINIAOD/PromptReco-v1/000/380/306/00000/1cfada9f-5e3e-4f4c-b38b-f55455e2e9f5.root` is now available. Looks like it was never cached and we were always reading it from `root://xrootd-cms.infn.it` . I will check why it was not cached

Thanks a lot @smuzaffar !

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**